CHANGES

```diff
@@ -14,6 +14,14 @@ The rules for this file:
 
 ------------------------------------------------------------------------------
 
+**/**/2024 xiki-tempula
+
+  * 2.2.1
+
```

Comment:
document default change
```suggestion

Changes
  - Default value for keyword argument `initial_nk` of the MBAR estimator was
    changed to "BAR" (run an initial BAR calculation before MBAR) instead of 
    `None` (start from all zeros) as this change provides a sizable speedup (PR #357)
    
```